src/components/Label/Label.tsx

```diff
@@ -53,13 +56,42 @@ const Label: FC<LabelProps> & {
           [styles.withAnimation]: !isAnimationDisabled,
           [styles.withLeg]: isLegIncluded
         },
-        labelClassName
+        labelClassName,
+        { [styles.clickable]: isClickable }
       ),
-    [kind, color, isAnimationDisabled, isLegIncluded, labelClassName]
+    [kind, color, isAnimationDisabled, isLegIncluded, labelClassName, isClickable]
   );
+
+  const onClickCallback = useCallback(
+    (event: React.MouseEvent<HTMLSpanElement, MouseEvent>) => {
+      if (onClick) {
+        event.preventDefault();
+        onClick(event);
+      }
+    },
+    [onClick]
+  );
```

Comment:
Why not do this inside `useClickableProps`?

Reply:
What do you mean?

Reply:
Oh you mean always preventDefault inside `useClickableProps`?